### ISSUE TITLE
Trying to embedd in docusaurus v2 and get the following errors:

### ISSUE THREAD

**Issue by mwawrusch** [apollographql/embeddable-explorer#166]
Seems there might be missing dependency in the package:

```
Compiled with problems:X

ERROR in ./node_modules/@apollo/explorer/EmbeddedExplorer-a17b5871.js 2:0-47

Module not found: Error: Can't resolve 'zen-observable-ts' in '/Users/martinwawrusch/Dev/roji/developer-roji-io/node_modules/@apollo/explorer'


ERROR in ./node_modules/@apollo/explorer/react/index.mjs 3:0-59

Module not found: Error: Can't resolve 'use-deep-compare-effect' in '/Users/martinwawrusch/Dev/roji/developer-roji-io/node_modules/@apollo/explorer/react'


ERROR in ./node_modules/@apollo/explorer/react/index.mjs 5:0-27

Module not found: Error: Can't resolve 'zen-observable-ts' in '/Users/martinwawrusch/Dev/roji/developer-roji-io/node_modules/@apollo/explorer/react'

```

**Comment by mayakoneval** [comment]
Hi! Thank you for the report. `zen-observable-ts` is in the wrong place. I have a PR up here you can track: https://github.com/apollographql/embeddable-explorer/pull/180

For `use-deep-compare-effect`, this is an [optional dependency for the explorer package](https://github.com/apollographql/embeddable-explorer/blob/main/packages/explorer/package.json#L67-L69). If you are using the react component, you will need to install it.

**Comment by mayakoneval** [comment]
Hi this should be fixed in @apollo/explorer@2.0.1 and @apollo/sandbox@1.0.2
